output/react/package.json

```
@@ -14,13 +14,15 @@
     "mv:dist": "cpr dist ../../build-outputs/react/dist -o",
     "mv:package.json": "cpr package.json ../../build-outputs/react/package.json -o",
     "mv:readme": "cpr README.md ../../build-outputs/react/README.md -o",
-    "postbuild": "npm-run-all -p mv:*"
+    "postbuild": "npm-run-all -p mv:*",
+    "test-ct": "playwright test -c playwright.config.ts"
```

```suggestion
    "test:ct": "playwright test -c playwright.config.ts"
```

what does `ct` stand for? Probably it's better to use the long form / name in here instead of the abbreviation for clarity.
Plus it seems to me like we could align the syntax from hyphen to colon.

---

Fixed